utils/osdns/dnsconf.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
```

Comment:
It looks like this file should also have the Go license in it.

It would be also good to point to the file it's based on.